test/cli/shared/update.dart

```diff
@@ -66,16 +66,19 @@ void sharedTests(Future<TestProcess> runSass(Iterable<String> arguments)) {
       await d.file("test2.scss", r"$var: 2; @import 'other'").create();
 
       var sass = await update(["test1.scss:out1.css", "test2.scss:out2.css"]);
-      expect(sass.stdout, emits(endsWith('Compiled test1.scss to out1.css.')));
-      expect(sass.stdout, emits(endsWith('Compiled test2.scss to out2.css.')));
```

Comment:
If the issue here is that the order is now non-deterministic, you can use [`emitsInAnyOrder`](https://pub.dev/documentation/test/latest/expect/emitsInAnyOrder.html) to test for it without mandating a specific order.